src/App.tsx

```diff
@@ -1,14 +1,57 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [loader, setLoader] = useState(false);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | ''>('');
+
+  const [todoFilter, setTodoFilter] = useState({
+    input: '',
+    filter: 'all',
+  });
+
+  useEffect(() => {
+    setLoader(true);
+
+    getTodos().then(data => {
+      setTodos(data);
+      setLoader(false);
+    });
+  }, []);
+
+  const currentTodos = useCallback(() => {
+    const { input, filter } = todoFilter;
+
+    if (!input && filter === 'all') {
+      return todos;
+    }
+
+    return todos.filter(content => {
+      if (input && !content.title.toLowerCase().includes(input.toLowerCase())) {
+        return false;
+      }
+
+      switch (filter) {
+        case 'active':
+          return !content.completed;
+        case 'completed':
+          return content.completed;
+        default:
+          return true;
```

Comment:
The `default` case in the switch statement now correctly returns `true`, which aligns with the expected behavior.